src/StreetNameRegistry.Api.BackOffice/Infrastructure/Authorization/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace StreetNameRegistry.Api.BackOffice.Infrastructure.Authorization
+{
+    using System.Collections.Generic;
+    using Microsoft.Extensions.DependencyInjection;
+    using NisCodeService.DynamoDb.Extensions;
+    using NisCodeService.HardCoded.Extensions;
+
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddNisCodeAuthorizationWithDynamoDb(this IServiceCollection services, IList<string>? ovoCodeWhiteList)
```

Comment:
AddRegistryNisCodeFinder should be called to add multiple RegistryNisCodeFinders, however this method does not force the user to register at least one RegistryNisCodeFinder, should be force it here by asking for one in the signature?

Reply:
I wouldn't necessarily do that.